files/lib/data/conversation/ConversationAction.class.php

```diff
@@ -74,8 +74,10 @@ public function create() {
 			// save participants
 			$conversationEditor->updateParticipants((!empty($this->parameters['participants']) ? $this->parameters['participants'] : []), (!empty($this->parameters['invisibleParticipants']) ? $this->parameters['invisibleParticipants'] : []));
 			
-			// add author
-			$conversationEditor->updateParticipants([$data['userID']]);
+			if($data['userID'] !== null) {
```

Comment:
I was about to hit merge, until that missing space between `if` and `(` came to my attention. You've added 3 lines of code with 1 line breaking the code style guidelines, that results in a 33% error quote ;)

Reply:
Can't edit it so ignore it and take the other one^^
Sry, but deleted the other branch directly after pushing the changes...

Reply:
@Morik You should be able to recover deleted branches using `git reflog`: https://git-scm.com/docs/git-reflog.

And in any case: Pull requests support force pushes, you can just perform a forced push into the remote branch and the pull request would update. Editing the existing commit and doing a force push would be preferred anyway, because it does not clutter the commit history.

Reply:
OK, i'll try this next time, sry for the mess.
But i'm using GitKraken as GUI for git and not the commandline, so i'm not that familiar with the real options i could use.

Reply:
@Morik While I personally don't use GUIs for git, but I see the charm for many. Unfortunately it's like those server administration tools: When everything is on fire they are simply inadequate.

Here's a tutorial to the use of the reflog: https://gist.github.com/jbgo/1944238

Reply:
I know, and under Linux i love the console.
But at home i'm a windows user and this includes programming for hobby projects while using klicki bunti GUIs.